frontend/src/scenes/session-recordings/player/playerUtils.ts

```diff
@@ -109,11 +109,33 @@ export function getPlayerPositionFromPlayerTime(
     return null
 }
 
+// Gets the player position from an epoch-time without a window-id
+// Used to place backend events in a recording even though, they don't
+// have a window id
+export function guessPlayerPositionFromEpochTimeWithoutWindowId(
+    epochTime: number,
+    startAndEndTimesByWindowId: Record<string, RecordingStartAndEndTime>,
+    segments: RecordingSegment[]
+): PlayerPosition | null {
+    for (const segment of segments) {
+        if (epochTime >= segment.startTimeEpochMs && epochTime <= segment.endTimeEpochMs) {
+            return getPlayerPositionFromEpochTime(epochTime, segment.windowId, startAndEndTimesByWindowId)
+        }
+    }
+    return null
+}
+
 export function getPlayerPositionFromEpochTime(
     epochTime: number,
     windowId: string,
     startAndEndTimesByWindowId: Record<string, RecordingStartAndEndTime>
 ): PlayerPosition | null {
+    if (!windowId && !startAndEndTimesByWindowId['']) {
```

Comment:
What's supposed to happen here?

Reply:
Gahh, good catch. That wasn't supposed to be there... this case is handled in the `sessionRecordingLogic`. I had briefly considered putting the logic in the util, but decided against it.